types/testingutils/network.go

```diff
@@ -10,6 +10,7 @@ type TestingNetwork struct {
 	BroadcastedMsgs []*types.SignedSSVMessage
 	OperatorID      types.OperatorID
 	OperatorSK      *rsa.PrivateKey
+	Domain          types.DomainType
```

Comment:
In the `NewTestingNetwork` function, this field is unassigned. I believe you could initialise it as a TestingSSVDomaintType. However, it's strange to me that the tests are still passing even though this is unassigned.